### ISSUE TITLE
Spotlight accessibility issues

### ISSUE THREAD

**Issue by LachlanMcKee** [bumble-tech/appyx#558]
The Spotlight component does not work correctly with accessibility.

Please review the following video.

https://github.com/bumble-tech/appyx/assets/1333155/458a2634-93d3-423d-8324-a49b611e1e92

### Issue 1
Using the single swipe gesture, a user should be able to navigate through the Spotlight elements. However instead the single swipe gesture is only navigating between what is visible on the screen.

It is only when I use a two finger gesture that this works (though it is not discoverable)

### Issue 2
The spotlight component does not indicate to the accessibility service that this is effectively a list of elements. It should populate the accessibility node with the current index, and the number of elements.

In doing so, talkback would indicate at the end "showing items 1 of 10" as an example
